templates/search/results-did-you-mean-fr.html

```diff
@@ -0,0 +1,33 @@
+---
+{
+	"title": "Résultats de la recherche (voulez-vous dire)",
+	"language": "fr",
+	"altLangPage": "results-did-you-mean-en.html",
+	"pageType": "recherche",
+	"pageclass": "page-type-search",
+	"dateModified": "2025-07-25",
+	"share": "false",
+	"deptfeature": false
+}
+---
+<form class="mrgn-tp-sm" role="search">
+	<div class="input-group mrgn-tp-lg">
+		<label class="wb-inv" for="sch-inp-ac">Résultats de la recherche (base) pour le gouvernement du Canada avec Headless</label>
+		<input id="sch-inp-ac" class="form-control" value="" name="q" autocomplete="off" spellcheck="false" type="search" aria-describedby="gc-pi">
+		<span class="input-group-btn">
+			<button class="btn btn-primary btn-small" type="submit"> <span class="glyphicon-search glyphicon" aria-hidden="true"></span> <span class="wb-inv">Search</span> </button>
+		</span>
+	</div>
+	<p id="gc-pi" class="mrgn-tp-md">N'incluez pas de renseignements personnels (téléphone, courriel, NAS, renseignements financiers, médicaux ou professionnels).</p>
+</form>
+
+<section id="wb-land">
+	<h2>["Query summary" or "No results" heading structure]</h2>
```

Comment:
Just a French translation:

```suggestion
	<h2>[Structure de titre pour "Résumé de requête" ou "Aucun résultat"]</h2>
```